SampleScripts/PowerManagement-WorkloadDomain.ps1

```diff
@@ -671,16 +710,47 @@ Try {
                             Exit
                         }
                     }
-                    #VSAN shutdown wizard automation
-                    Set-VsanClusterPowerStatus -server $vcServer.fqdn -user $vcUser -pass $vcPass -cluster $cluster.name -PowerStatus clusterPoweredOff
-                    $esxiDetails = $esxiWorkloadCluster[$cluster.name]
-                    foreach ($esxiNode in $esxiDetails) {
-                        if ((Test-NetConnection -ComputerName $esxiNode.fqdn -Port 443).TcpTestSucceeded) {
-                            Write-PowerManagementLogMessage -Type ERROR -Message "$($esxiNode.fqdn) is still up. Check the FQDN or IP address, or the power state of the '$($esxiNode.fqdn)'." -Colour Red
-                            Exit
+					$esxiDetails = $esxiWorkloadCluster[$cluster.name]
+					
+                    #VSAN shutdown wizard automation or VxRail shutdown wizard automation
```

Comment:
```suggestion
                    # vSAN or VxRail Manger shutdown wizard automation.
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108